docs/unit-testing-a-new-schema.md

```diff
@@ -0,0 +1,69 @@
+# Unit testing a new schema
+
+## Overview
+As the chs-delta-api uses a 3rd party library (See `/docs` directory for documentation on the kin-openAPI3 library) to 
+validate its requests, there is no way to directly unit test the code. This document will guide you through creating 
+unit tests which assert that your open API schema is working correctly.
+
+## 1. Where to add your unit tests
+Move into the `/validation/schema_testing` directory and create a new directory to contain your unit tests. Inside of your
+new directory, create a set `/request_bodies` and `/response_bodies` directories. These will contain your sample request 
+and response bodies used to unit test the schema.
+
+Finally, create a new Go file ending in `_test` (e.g. `exampleDeltaSchema_test.go`). This file will contain the unit 
+tests used to test the schema.
+
+## 2. Unit test structure
+We elected to structure the open API schema unit tests using a `Karate API Scenario` layout. Each unit test will have a 
```

Comment:
As Tom suggested previously, OpenApi is one word

Reply:
Changed this